template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/efm/EfmRoleConfigProvider.java

```diff
@@ -0,0 +1,56 @@
+package com.sequenceiq.cloudbreak.cmtemplate.configproviders.efm;
+
+import static com.sequenceiq.cloudbreak.cmtemplate.configproviders.ConfigUtils.config;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.stereotype.Component;
+
+import com.cloudera.api.swagger.model.ApiClusterTemplateConfig;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.database.base.DatabaseType;
+import com.sequenceiq.cloudbreak.cmtemplate.configproviders.AbstractRdsRoleConfigProvider;
+import com.sequenceiq.cloudbreak.template.TemplatePreparationObject;
+import com.sequenceiq.cloudbreak.template.views.RdsView;
+
+@Component
+public class EfmRoleConfigProvider extends AbstractRdsRoleConfigProvider {
+
+    static final String DATABASE_USER = "efm.db.username";
+
+    static final String DATABASE_PASSWORD = "efm.db.password";
+
+    static final String DATABASE_URL = "efm.db.url";
+
+    static final String DRIVER_CLASS = "efm.db.driverClass";
+
+    @Override
+    protected List<ApiClusterTemplateConfig> getRoleConfigs(String roleType, TemplatePreparationObject source) {
+        RdsView efmRdsView = getRdsView(source);
+        List<ApiClusterTemplateConfig> configs = new ArrayList<>();
+
+        if (EfmRoles.EFM_SERVER.equals(roleType)) {
+            configs.add(config(DATABASE_URL, efmRdsView.getConnectionURL()));
```

Comment:
Looks alright. Just a quick q: Which version of the PostgreSQL JDBC driver is being included / used? Asking for sake of SSL mode.

Context: When applicable, SSL mode options will be automatically included in `getConnectionURL()`, but the JDBC driver should be recent enough (>=42.2.5) in order to correctly interpret the DB server certificate chain. We raised a related bug ticket against NiFi Registry a couple of weeks ago; turned out they rely on some ancient JDBC driver that cannot cope with the Subject Alternative Name in the certificate chain. See [CFM-3228](https://jira.cloudera.com/browse/CFM-3228) if interested.